products/slackware.md

```diff
@@ -0,0 +1,109 @@
+releases:
+-   releaseCycle: "current"
+    codename: "-current"
+    eol: false
+    link: http://www.slackware.com/changelog/current.php?cpu=i386
+    releaseDate: 1993-07-27
+-   releaseCycle: "15.0"
+    codename: "stable"
+    eol: false
+    link: http://www.slackware.com/announce/15.0.php
+    releaseDate: 2022-02-02
+-   releaseCycle: "14.2"
+    eol: false
+    link: http://www.slackware.com/announce/14.2.php
+    releaseDate: 2016-06-30
+-   releaseCycle: "14.1"
+    eol: false
+    link: http://www.slackware.com/announce/14.1.php
+    releaseDate: 2013-11-04
+-   releaseCycle: "14.0"
+    eol: false
+    link: http://www.slackware.com/announce/14.0.php
+    releaseDate: 20l2-09-26
+-   releaseCycle: "13.37"
+    eol: 2018-07-05
+    link: http://www.slackware.com/announce/13.37.php
+    releaseDate: 2011-04-25
+-   releaseCycle: "13.1"
+    eol: 2018-07-05
+    link: http://www.slackware.com/announce/13.1.php
+    releaseDate: 2010-05-19
+-   releaseCycle: "13.0"
+    eol: 2018-07-05
+    link: http://www.slackware.com/announce/13.0.php
+    releaseDate: 2009-08-26
+-   releaseCycle: "12.2"
+    eol: 2013-12-09
+    link: http://www.slackware.com/announce/12.2.php
+    releaseDate: 2008-12-09
+-   releaseCycle: "12.1"
+    eol: 2013-12-09
+    link: http://www.slackware.com/announce/12.1.php
+    releaseDate: 2008-05-01
+-   releaseCycle: "12.0"
+    eol: 2012-08-01
+    link: http://www.slackware.com/announce/12.0.php
+    releaseDate: 2007-07-01
+-   releaseCycle: "11.0"
+    eol: 2012-08-01
+    link: http://www.slackware.com/announce/12.0.php
+    releaseDate: 2006-10-01
+-   releaseCycle: "10.2"
+    eol: 2012-08-01
+    link: http://www.slackware.com/announce/12.0.php
+    releaseDate: 2005-09-13
+-   releaseCycle: "10.1"
+    eol: 2012-08-01
+    link: http://www.slackware.com/announce/12.0.php
+    releaseDate: 2005-02-02
+-   releaseCycle: "10.0"
+    eol: 2012-08-01
+    link: http://www.slackware.com/announce/12.0.php
+    releaseDate: 2004-06-22
+-   releaseCycle: "9.1"
+    eol: 2012-08-01
+    link: http://www.slackware.com/announce/12.0.php
+    releaseDate: 2003-09-25
+-   releaseCycle: "9.0"
+    eol: 2012-08-01
+    link: http://www.slackware.com/announce/12.0.php
+    releaseDate: 2003-03-18
+-   releaseCycle: "8.1"
+    eol: 2012-08-01
+    link: http://www.slackware.com/announce/12.0.php
+    releaseDate: 2002-06-19
+-   releaseCycle: "8.0"
+    eol: true
+    link: http://www.slackware.com/announce/12.0.php
+    releaseDate: 2001-06-28
+
+---
+
+> [Slackware](http://www.slackware.com/) is a general-purpose GNU/Linux distribution that aims to be most ["UNIX-like", simple and stable](http://www.slackware.com/info/).  The developers strive to minimize the number of changes to upstream software relases.  It is the oldest actively developed GNU/Linux distribution with development being led by creator Patrick Volkerding continuously since 1993.
```

Comment:
```suggestion
> [Slackware](http://www.slackware.com/) is a general-purpose GNU/Linux distribution that aims to be most ["UNIX-like", simple and stable](http://www.slackware.com/info/). It strives to minimize the number of changes to upstream software releases.  It is the oldest actively developed GNU/Linux distribution with development being led by creator Patrick Volkerding continuously since 1993.
```